extensions/ql-vscode/src/run-queries.ts

```diff
@@ -216,6 +218,29 @@ export class QueryInfo {
     return this.dilPath;
   }
 
+  async exportCsvResults(qs: qsClient.QueryServerClient, csvPath: string, onFinish: () => void): Promise<void> {
+    let stopDecoding = false;
+    const out = fs.createWriteStream(csvPath);
```

Comment:
If the csv file already exists, then maybe no need to recreate it? Though if the operation is fast, not a big deal either way.

Reply:
This function is already not being called if the file already exists (see [this line](https://github.com/github/vscode-codeql/blob/6564196328680cecee581353b80fa3e8e30b0ec5/extensions/ql-vscode/src/query-history.ts#L587))